python/cuml/test/test_dbscan.py

```diff
@@ -137,3 +137,104 @@ def test_dbscan_out_dtype_fails_invalid_input():
 
     cudbscan = cuDBSCAN(output_type='numpy')
     cudbscan.fit_predict(X, out_dtype="bad_input")
+
+
+def test_core_point_prop1():
+    params = {'eps': 1.1, 'min_samples': 4}
```

Comment:
Were these tests able to reproduce the core points bug here? They seem pretty simple and it would be nice to see a simple gtest that could catch this issue in the future.

Reply:
Yes exactly, you can run these tests on the current implementation (without this PR) and they should fail. These tests are specifically designed to fail the current implementation.
And I agree with you, it was a bit faster to code these in Python for me, as I can see the data in space in PyCharm, so that's why I put them there, but they should ultimately go into `cpp/test/sg/dbscan_test.cu`, right?

Reply:
yes, these should end in dbscan_test.cu